ratis-metrics/src/main/java/org/apache/ratis/metrics/MetricRegistries.java

```diff
@@ -0,0 +1,86 @@
+/**
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ratis.metrics;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.Set;
+
+import com.codahale.metrics.MetricRegistry;
+import org.apache.ratis.metrics.impl.RatisMetricRegistry;
+
+/**
+ * MetricRegistries is collection of MetricRegistry's. MetricsRegistries implementations should do
+ * ref-counting of MetricRegistry's via create() and remove() methods.
+ */
+public abstract class MetricRegistries {
```

Comment:
cc/ @jnp

What's your take on keeping a ratis-metrics module? You had expressed offline that you thought wrapping dropwizard metrics classes was overkill -- do you think having this "entrypoint" for Ratis to get/access metrics is OK?

I think it's still good (would help us keep some stability).